pages/explorer/dps.tsx

```diff
@@ -52,8 +55,28 @@ const DpsPage: NextPage<DpsProps> = ({
     ebpsData: ebpsData,
   };
 
+  const keywords = generateKeywordsString([
+    "coh3 dps",
+    "dps tools",
+    "coh3 units calculator",
+    "coh3 damage calculator",
+    "damage per second coh3",
+  ]);
+
   return (
     <>
+      <Head>
+        <title>COH3 Stats - DPS Calculator </title>
+        <meta
+          name="description"
+          content={
+            "DPS Calculator for all units. Calculate with types of covers. " +
```

Comment:
@chris-nos  this adds description and tags for the dps page, feel free to update them and polish them 👍